nix/haskell.nix

```diff
@@ -53,7 +53,8 @@ let
           "hpc"
           "mtl" "parsec" "text" "transformers"
           "xhtml"
-          # "stm" "terminfo"
+          # "stm"
+          "terminfo"
```

Comment:
I now think this only has effect on Windows (see the `isWindows` on line 34), although my understanding of Nix is limited.

Reply:
Alternatively, we just wait for #1977 to be merged, as it will remove the dependency on `vty` (which relies on `terminfo`) :grin:.

Reply:
@mrBliss, your change should be sufficient -- `isWindows` only applies to the first, small attrset:
https://github.com/input-output-hk/cardano-node/blob/70d6ab9ceae2fe9929b640ec6b066543d4584e6a/nix/haskell.nix#L34-L38